ecs-init/exec/iptables/iptables.go

```diff
@@ -69,6 +78,11 @@ func NewNetfilterRoute(cmdExec exec.Exec) (*NetfilterRoute, error) {
 		return nil, err
 	}
 
+	defaultOffhostIntrospectionInterface, err = getOffhostIntrospectionInterface()
```

Comment:
I'm wondering if we should have a fallback here for if this fails? Do we know for sure that we'll always have access to `/proc/net/route` and it will always look as expected here?

Reply:
This file is created by kernel at boot time as part of procfs so the content should be kernel's knowledge of IPv4 routes. I'm not sure if there should be a fallback or what a reasonable fallback would be if we couldn't find a default route there. 

I checked latest AL1, AL2, AL2022, Ubuntu 20.04, SUSE SLES 15, and Debian 10, all have the file present, readable by everyone (permission 444), and contain the default route. Any suggestion on other distros I should check?

Reply:
That seems like a reasonable range of distros to test. I tried to find some other distro with a different file but it does seem fairly consistent